tidb-cloud/integrate-tidbcloud-with-vercel.md

```diff
@@ -145,19 +151,74 @@ The detailed steps are as follows:
 </div>
 </SimpleTab>
 
+### Configure connections
+
+If you have installed [TiDB Cloud Vercel integration](https://vercel.com/integrations/tidb-cloud), you can add or remove connections inside the integration.
+
+1. In your Vercel dashboard, click **Integrations**.
+2. Click **Manage** in the TiDB Cloud entry.
+3. Click **Configure**.
+4. Click **Add Link** or **Remove** to add or remove connections.
+
+    ![Vercel Integration Configuration Page](/media/tidb-cloud/vercel/integration-vercel-configuration-page.png)
+
+    When you remove a connection, the environment variables set by the integration workflow are removed from the Vercel project, too. However, this action does not affect the data of the TiDB Serverless cluster.
+
+### Connect with TiDB Serverless branching
+
+Vercel's [Preview Deployments](https://vercel.com/docs/deployments/preview-deployments) feature allows you to preview changes to your app in a live deployment without merging those changes to your Git project's production branch. With [TiDB Serverless Branching](/tidb-cloud/branch-overview.md), you can create a new instance for each branch of your Vercel project. This allows you to preview app changes in a live deployment without affecting your production data.
+
+> **Note:**
+>
+> Currently, TiDB Serverless branching only supports [Vercel projects associated with GitHub repositories](https://vercel.com/docs/deployments/git/vercel-for-github).
+
+To enable TiDB Serverless Branching, you need to:
+
+1. Select **Cluster** as your connection type. 
+2. Enable TiDB Serverless Branching in the TiDB Cloud Vercel integration workflow.
+
+After you push changes to GitHub repository, Vercel will trigger a preview deployment. And then TiDB Cloud integration will automatically create a TiDB Serverless branch for the branch and set environment variables. The detailed steps are as follows:
+
+1. Create a new branch in your Git repository.
```

Comment:
```suggestion
1. Create a new branch in your GitHub repository.
```

Reply:
I think `Git` is deliberately used here, not `GitHub`. `Git` means the VCS tech while `GitHub` means the VCS vendor, not the same.

Reply:
Yes, we should use Git here